experiments/20160805-mutagenesis/20160928-cpec-pol-assay

```diff
@@ -0,0 +1,27 @@
+## Goals
+
+- Identify polymerase plasmids with successfully mutated PB2 part 2 fragment from [CPEC library](./201609260transformation.md) for sequencing
+- Expecting luminescence results for mutants to be distinct from results for the template
+
+### Cell culture
+
+Followed [cell culture protocol](protocols/cell-culture.md) for preparing for polymerase assay.
+Cells were grown in 1:10 concentration, and used at 80% confluence after one day incubation period.
+
+### Transfection
+
+Followed [transfection protocol](protocols/transfection.md) using the following testing setup.

```

Comment:
Likewise here for the correct relative path. Does GitHub render them correctly without the "`../../`"?